.pre-commit-config.yaml

```
@@ -49,33 +49,33 @@ repos:
       - id: detect-private-key
 
   - repo: https://github.com/asottile/pyupgrade
-    rev: v2.34.0
+    rev: v3.3.1
     hooks:
       - id: pyupgrade
         args: [--py37-plus]
         name: Upgrade code
 
-  - repo: https://github.com/myint/docformatter
-    rev: v1.4
+  - repo: https://github.com/PyCQA/docformatter
+    rev: v1.5.1
```

There are issues with this version. I suggest leaving this to a later PR

```suggestion
    rev: v1.4
```

---

yes, we skipped 1.5 as there were too many unnecessary changes :)